### PR TITLE
Created an actual table [Manual,Scope of Variables]

### PR DESCRIPTION
**Pull request by ghost**
It looks clearer to me, by showing immediately the *constructs* the document talks about. 
In the the original tree , a complete beginner may confuse which are the constructs, and which are the types of scope

Also removed the mention "(don't allow nesting)" in the 5th branch of the tree, as that is already stated by the 1st branch of the tree
(It had actually confused me https://discourse.julialang.org/t/scope-of-struct-and-macro/16714 )

### CODE REVIEW

**Comment by VictorelPetrovich** [comment]
I've no idea what those "checks" above mean, I'm just editing the documentation, by following the "Edit on Github" link from https://docs.julialang.org/en/v1/manual/variables-and-scoping/#man-scope-table-1

**Comment by ararslan** [comment]
Welcome and thanks for the contribution!

If you click the "Details" link next to each of the continuous integration checks, you can see a log for the job that ran the tests. In this case, they're failing because your changed introduced trailing whitespace. You can fix that by deleting the spaces at the end of the line on lines 24 and 25 of the file.

**Comment by VictorelPetrovich** [comment]
Thanks @ararslan ! 
Why it's taking so long for Travis CI building process? ( 5 min and still counting). Is it building the entire Julia implementation, or just the whole  documentation?

**Comment by fredrikekre** [comment]
It needs to build Julia in order to build the docs.

**Comment by KristofferC** [comment]
> I think there should be leaner ways to check doc changes only

The doctests need to be run and to do that you need to have Julia built to run it.

**Comment by VictorelPetrovich** [comment]
If the changes only affect documentation, cannot you use an already built version of Julia to run the doctests?

**Comment by KristofferC** [comment]
No :). It would be more productive to focus on the content of the PR instead of how CI is done.

File: doc/src/manual/variables-and-scoping.md [hunk 1]
```diff
@@ -13,29 +13,16 @@ set of source lines; instead, it will always line up with one of these blocks. T
...
+### [Scope constructs](@id man-scope-table)
+
+Construct | Scope type | Scope blocks it may be nested in
+------------ | -------------  |---------------------------
+[`module`](@ref), [`baremodule`](@ref)            | global | other global
+interactive prompt (REPL)                         | global | other global
+(mutable) [`struct`](@ref), [`macro`](@ref)       | local  | other global
+[`for`](@ref), [`while`](@ref), [`try-catch-finally`](@ref try), [`let`](@ref) |local | other local
+functions (either syntax, anonymous & do-blocks) | local | other local
+comprehensions, broadcast-fusing                 | local | other local
```
Comment (fredrikekre) [inline doc/src/manual/variables-and-scoping.md]: The three last rows here should state `other local or global` as per the old docs I assume?

**Comment by VictorelPetrovich** [comment]
Ah, how could I miss that!! (but I _knew_ those 3 constructs can nest in either global or local scopes). Thanks for checking

File: doc/src/manual/variables-and-scoping.md (hunk 1 above)
Comment (VictorelPetrovich) [reply doc/src/manual/variables-and-scoping.md]: Ah, how could I miss that!! (but I knew those 3 constructs can nest in either global or local scopes). Thanks for checking

**Comment by VictorelPetrovich** [comment]
:clap:

**Pull request merged**